src/pages/components/modal/usage.mdx

```diff
@@ -368,11 +369,11 @@ errors or conveying a consequence of a user's action.
 
 Passive modals are persistent until dismissed in one of the following ways.
 
-- **x**: Clicking the close `x` icon in the upper right will close the modal
+- **x**: Clicking the close **x** icon in the upper right will close the modal
   without submitting any data and return the user to its previous context.
 - **Click elsewhere**: Clicking outside the passive modal area will
   automatically close the modal.
-- **Esc**: Press `ESC` on the keyboard
+- **Esc**: Pressing ESC on the keyboard
```

Comment:
We always put key names in code snippets on our site. It will be inconsistent across the pages and tabs if we start removing that style here.
```suggestion
- **Esc**: Pressing `ESC` on the keyboard
```